client/rpc/src/overrides/mod.rs

```diff
@@ -1,18 +1,21 @@
-// Copyright 2017-2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
 // This file is part of Frontier.
-
-// Substrate is free software: you can redistribute it and/or modify
+//
+// Copyright (c) 2020 Parity Technologies (UK) Ltd.
```

Comment:
I don't think this is right. Should be 2017-2020.

Any reason you change this copyright header format? If so, we also need to change all other headers as they are of the same format.

Reply:
The license headers of the code files in the current frontier repository are not uniform. My suggestion is to use the `license_template_path` option of rustfmt to unify it. If you don't mind, I can open another PR to do this.

Reply:
If `license_template_path` doesn't work yet, let's just fix the copyright header as we see it for now. I think the only thing needed for this PR for this is to restore the original years (2017-2020).

Reply:
okay